### PR TITLE
[0.26] first pass at silenced api docs (WIP)

### PR DESCRIPTION
**Pull request by cwjohnston**
First pass at documenting `/silenced` API to be introduced in 0.26.

### CODE REVIEW

File: docs/0.26/api/silenced-api.md [hunk 1]
```diff
@@ -0,0 +1,399 @@
...
+  - [`/silenced/checks/:check` (GET)](#silenced-checks-get)
+
+## The `/silenced` API endpoints
+
+The Silence API provides endpoint HTTP POST and GET access to create, query and
+clear (delete) a silencing override via the Sensu API.
+
+### `/silenced` (GET)
+
+#### Example: Querying for all silenced overrides
```
Comment (portertech) [inline docs/0.26/api/silenced-api.md]: "overrides" -> "entries"?

File: docs/0.26/api/silenced-api.md [hunk 2]
```diff
@@ -0,0 +1,399 @@
...
+  - [`/silenced` (GET)](#silenced-get)
+  - [`/silenced` (POST)](#silenced-post)
+  - [`/silenced/clear` (POST)](#silenced-clear-post)
+  - [`/silenced/subscriptions/:subscription` (GET)](#silenced-subscriptions-get)
+  - [`/silenced/checks/:check` (GET)](#silenced-checks-get)
+
+## The `/silenced` API endpoints
+
+The Silence API provides endpoint HTTP POST and GET access to create, query and
+clear (delete) a silencing override via the Sensu API.
```
Comment (portertech) [inline docs/0.26/api/silenced-api.md]: "a silence entry"

File: docs/0.26/api/silenced-api.md [hunk 3]
```diff
@@ -0,0 +1,399 @@
...
+#### Example: Creating a silence entry {#silence-post-examples}
+
+The following example demonstrates a `/silenced` query, which creates a
+silence entry for the check "check_haproxy" on clients with the
+"load-balancer" subscription, with an expiration of 3600 seconds:
+
+~~~ shell
+$ curl -s -i -X POST \
+-H 'Content-Type: application/json' \
+-d '{"subscription": "load-balancer", "check": "check_haproxy", "expires": 3600 }' \
```
Comment (portertech) [inline docs/0.26/api/silenced-api.md]: should be "expire"

File: docs/0.26/api/silenced-api.md [hunk 4]
```diff
@@ -0,0 +1,399 @@
...
+  : - **Success**: 201 (Created)
+    - **Malformed**: 400 (Bad Request)
+    - **Error**: 500 (Internal Server Error)
+
+
+### `/silenced/clear` (POST)
+
+#### Example: Clearing a silence entry
+
+A silence entry can be deleted by its ID:
```
Comment (portertech) [inline docs/0.26/api/silenced-api.md]: "can be cleared"?

File: docs/0.26/reference/events.md [hunk 5]
```diff
@@ -151,6 +151,18 @@ document:
     "occurrences": 3
     ~~~
 
+`occurrences_watermark`
```
Comment (calebhailey) [inline docs/0.26/reference/events.md:154]: was this supposed to be in this PR? 

...says the worst offender of all time at including unrelated commits in PRs...

**Comment by calebhailey** [comment]
This is looking great to me – lots of excellent examples, which will be helpful as users begin to use this. :+1:

File: docs/0.26/reference/events.md (hunk 5 above)
Comment (cwjohnston) [inline docs/0.26/reference/events.md:154]: `occurrences_watermark` isn't part of the silencing implementation, but I did add it here intentionally as it needed documentation and it is also added in 0.26. good catch though 😉
